src/agent/driver-test-runner.js

```diff
@@ -135,8 +153,31 @@ export class DriverTestRunner {
       }
     }
 
+    const testId = test.info.testId;
+
+    if (this.callbackUrl) {
+      const headers = {
+        'Content-Type': 'application/json',
+      };
+      if (this.callbackHeader) {
+        const [name, value] = this.callbackHeader.split(/:\s*/);
```

Comment:
Since this is intended to hold a secret, it seems possible that the value will include the colon character. However, if we split like this, we'll end up truncating the secret in those cases.

Reply:
I wasn't 100% sure we wanted to use the header named `Secret:` - so put in a single split here, two ways to solve it:

1) Hard code the header name (not a terrible option, just want to know for sure what the aria-at-app is going to be looking for)
2) Change the split to only match the first `/:\s+/` in the header.